### ISSUE TITLE
`itertools.count` docstring is not shown correctly

### ISSUE THREAD

**Issue by JelleZijlstra** [microsoft/pylance-release#3231]
## Environment data

-   Language Server version: Pylance 2022.8.30
-   OS and version: macOS 11.6.8
-   Python version (& distribution if applicable, e.g. Anaconda): Python 3.10.2

## Code Snippet

```python
import itertools
itertools.count(1000)
```

## Repro Steps

1. Hover over `itertools.count`

## Actual behavior

<img width="523" alt="Screen Shot 2022-08-22 at 2 59 34 PM" src="https://user-images.githubusercontent.com/906600/186026881-c6a66b3e-4e16-4054-bc89-46aae1dd9203.png">

Notice how several lines in the code sample are mushed together.

For reference the full docstring is:

```
Return a count object whose .__next__() method returns consecutive values.

Equivalent to:
    def count(firstval=0, step=1):
        x = firstval
        while 1:
            yield x
            x += step
```

I suppose this is because the docstring is interpreted as Markdown and therefore adjacent lines are merged.

**Comment by judej** [comment]
Although there is no marker for code, we could watch for indentation to determine if it is code.

**Comment by Eureka-0** [comment]
Can I change the font style for this information? I noticed it to use the same font as my editor when using jedi.

pylance, not monospace and too small, hard to read
<img width="673" alt="iShot_2022-09-02_10 00 52" src="https://user-images.githubusercontent.com/56858288/188043102-f585e80c-9b36-460e-8624-41931b7dccb1.png">

jedi, the same font as editor, easy to read
<img width="548" alt="iShot_2022-09-02_10 01 38" src="https://user-images.githubusercontent.com/56858288/188043250-e5b78a89-d86b-40dd-9277-b6b5da75b852.png">

**Comment by debonte** [comment]
@Eureka-0, please open a separate issue on this.

**Comment by Eureka-0** [comment]
> @Eureka-0, please open a separate issue on this.

Ok I opened a new issue here #3278.

**Comment by rchiodo** [comment]
This issue has been fixed in prerelease version 2024.6.100, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#20246100-13-June-2024-PreRelease)
